tests/icon/index.test.tsx

```diff
@@ -93,12 +94,22 @@ describe('Icon', () => {
     icon.simulate('mouseenter');
     expect(onVisibleChange).toHaveBeenCalledWith(true);
     await sleep(0);
-    expect(wrapper.find('.ant-tooltip').at(1).hasClass('ant-tooltip-hidden')).toBe(false);
+    expect(
+      wrapper
+        .find('.ant-tooltip')
+        .at(0)
```

Comment:
I don't understand why previous test case pass  

Looks like we nedd to get first tooltip dom, according to https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/at.html  it should be 0